containers/proxy-httpd-image/Dockerfile

```diff
@@ -1,18 +1,41 @@
 #!BuildTag: uyuni/proxy-httpd:latest uyuni/proxy-httpd:%PKG_VERSION% uyuni/proxy-httpd:%PKG_VERSION%.%RELEASE%
 
 ARG BASE=registry.suse.com/bci/bci-base:15.3
-FROM $BASE
-
+FROM $BASE AS base
 
 ARG PRODUCT_REPO
+
+# Add distro and product repos
+COPY add_repos.sh /usr/bin
+RUN sh add_repos.sh ${PRODUCT_REPO}
+
+# Main packages
+RUN echo "rpm.install.excludedocs = yes" >>/etc/zypp/zypp.conf
```

Comment:
It's interesting you need to add this. The BCI image should already configure this by default. Otherwise it means a user of the bci-image would install the doc if he adds packages.

Maybe you can report this to the BCI maintainers?

Reply:
Thank you for your feedback :+1: Let me double check if it's something we really need to do. We've just brainstormed different strategies and I don't remember if it's really a problem

Reply:
Well yes 7.5M in `/usr/share/doc/packages` if called without `RUN echo "rpm.install.excludedocs = yes" >>/etc/zypp/zypp.conf`

Reply:
Yup, I am not saying you should remove this. Keep it in the PR.

But I think the BCI image should have this configured by default? I guess nobody wants doc by default when you install packages on containers? Can you check with them?

JeOS has the installation of doc disable, so I guess it should be the same for the BCI containers.

Reply:
I'll check with them, thank you!